.github/workflows/main.yml

```diff
@@ -28,7 +30,11 @@ jobs:
     - name: Config
       id: cfg
       run: |
-        if "${GITHUB_REF}" in "refs/tags/"*; then
+        push_test_pypi=no
+        push_pypi=no
```

Comment:
These two lines have no effect, they're setting environment variables that are never exported or used.